<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMainWindow</class>
 <widget class="QMainWindow" name="FormMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>irc2me</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="NetworkList" name="treeWidget_networklist">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="ChatView" name="chatView"/>
        </item>
        <item>
         <widget class="InputPrompt" name="widget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>670</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuServer">
    <property name="title">
     <string>&amp;Connection</string>
    </property>
    <addaction name="actionShow_connection_status"/>
    <addaction name="separator"/>
    <addaction name="action_Identities"/>
    <addaction name="action_Networks"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuServer"/>
  </widget>
  <action name="actionShow_connection_status">
   <property name="text">
    <string>&amp;Status</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_Networks">
   <property name="text">
    <string>&amp;Networks</string>
   </property>
  </action>
  <action name="action_Identities">
   <property name="text">
    <string>&amp;Identities</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NetworkList</class>
   <extends>QTreeWidget</extends>
   <header>widgets/networklist.h</header>
  </customwidget>
  <customwidget>
   <class>ChatView</class>
   <extends>QTableWidget</extends>
   <header>widgets/chatview.h</header>
  </customwidget>
  <customwidget>
   <class>InputPrompt</class>
   <extends>QWidget</extends>
   <header>widgets/inputprompt.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
